Makefile

```diff
@@ -170,6 +170,8 @@ check:
 	@! ineffassign . | grep -vF gossip/gossip.pb.go
 	@echo "errcheck"
 	@errcheck -ignore 'bytes:Write.*,io:Close,net:Close,net/http:Close,net/rpc:Close,os:Close,database/sql:Close' $(PKG)
+	@echo "returncheck"
+	@returncheck $(PKG)
```

Comment:
I do sometimes use `make PKG=./foo`, which behaves as you'd expect because
of this choice.

On Mon, Jan 25, 2016 at 11:34 AM, Peter Mattis <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/cockroachdb/cockroach/pull/3944#discussion_r50708503>:
>
> > @@ -170,6 +170,8 @@ check:
> >  	@! ineffassign . | grep -vF gossip/gossip.pb.go
> >  	@echo "errcheck"
> >  	@errcheck -ignore 'bytes:Write.*,io:Close,net:Close,net/http:Close,net/rpc:Close,os:Close,database/sql:Close' $(PKG)
> > +	@echo "returncheck"
> > +	@returncheck $(PKG)
>
> It's strange that this references PKG, but I see you are following the
> pattern of some of the other checks. Does anyone use make check PKG=./foo?
> I would change this and other checks to use ./.... If someone wants to
> run returncheck on a specific package they can do returncheck ./foo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/3944/files#r50708503>.
>